perfact/zodbsync/subcommand.py

```diff
@@ -139,6 +140,75 @@ def check_repo(self):
         # The commit to compare to with regards to changed files
         self.orig_commit = self.branches[self.orig_branch]
 
+    def _playback_paths(self, paths):
+        paths = self.sync.prepare_paths(paths)
+        dryrun = self.args.dry_run
+
+        playback_hook = self.config.get('playback_hook', None)
+        if playback_hook and os.path.isfile(playback_hook):
+            proc = subprocess.Popen(
+                playback_hook, stdin=subprocess.PIPE,
+                stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
```

Comment:
Didn't see that earlier - why do you capture `stderr`? I think it would be better if the caller would see any error messages on their command line.

Reply:
For now, I will merge this since I want to build on top of this. I think we should continue the discussion about what to to with `stderr` here, but that can be done in a later PR. For now, the hook script must not write anything to `stderr` in the case of success.